boost-common/src/main/java/io/openliberty/boost/utils/SpringBootUtil.java

```diff
@@ -159,6 +172,33 @@ public static Properties getSpringBootServerProperties(String buildDir) throws I
         String serverAddress = (String) allProperties.getOrDefault(SERVER_ADDRESS, DEFAULT_SERVER_ADDRESS);
         serverProperties.setProperty(SERVER_PORT, serverPort);
         serverProperties.setProperty(SERVER_ADDRESS, serverAddress);
+        
+        String serverSslKeystore = (String) allProperties.getProperty(SERVER_SSL_KEYSTORE);
+        String serverSslKeystorePassword = (String) allProperties.getProperty(SERVER_SSL_KEYSTORE_PASSWORD);
+        String serverSslKeystoreType = (String) allProperties.getProperty(SERVER_SSL_KEYSTORE_TYPE);
```

Comment:
No reason to get the properties from 178 to 181 until the **if** stmt at line 184 is checked.